### PR TITLE
Fix/math

### LINKED ISSUE #69: Math not showing properly
The mathematical components are not rendering properly, trying to use `react-katex` to resolve this.

### PR DESCRIPTION
**Pull request by acholyn**
Figuring out how best to display the mathematical components, mainly on the indices page.

Closes https://github.com/UCL/UKCORDEX-plot-explorer/issues/69

### CODE REVIEW

**Review by clairbarnes — APPROVED** [review]
All equations are now displaying correctly

File: src/components/ExtraPages/Help/IndicesHelp.js [hunk 1]
```diff
@@ -2,11 +2,12 @@ import React from "react";
...
 import "../ExtraPages.css";
 import "../HelpNav.css";
-import Latex from "react-latex";
+import { InlineMath, BlockMath } from "react-katex";
+import 'katex/dist/katex.min.css';
 import { HashLink } from "react-router-hash-link";
 
 function IndicesHelp() {
-  // var Latex = require("react-latex");
+  // var InlineMath = require("react-InlineMath");
```
Comment (raquelalegre) [inline src/components/ExtraPages/Help/IndicesHelp.js]:
```suggestion
```

File: src/components/ExtraPages/Help/IndicesHelp.js [hunk 2]
```diff
@@ -111,647 +112,778 @@ function IndicesHelp() {
...
-          <Row className="indices">
-            <h5 id="temp">
-              <u>Indices of temperature</u>
-            </h5>
-            <p className="foreword">
-              In defining the following indices of temperature, let tas_ij be
-              the daily mean temperature on day i in season j; the daily minimum
-              and maximum temperature on day i in season j are denoted by
-              tmin_ij and tmax_ij respectively.
+              <Row className="indices">
```
Comment (raquelalegre) [inline src/components/ExtraPages/Help/IndicesHelp.js]:
```suggestion
      <Row className="indices">
```

File: src/components/ExtraPages/Help/IndicesHelp.js [hunk 3]
```diff
@@ -111,647 +112,778 @@ function IndicesHelp() {
...
+      </p>{" "}
+      <p className="contents">
+        <b>Number of windy days</b>, <InlineMath math={"\\texttt{nwindy}"} />:
+        The number of windy days in season <InlineMath math={"j"} /> is the
+        count of days where{" "}
+        <InlineMath math={"\\texttt{sfcWind}_{ij} \\geq 10.8ms^{-1}"} />.
+      </p>
+    </Row>
           {/* extreme events */}
           <Row className="indices">
```
Comment (raquelalegre) [inline src/components/ExtraPages/Help/IndicesHelp.js]:
```suggestion
    <Row className="indices">
```

File: src/components/ExtraPages/Help/IndicesHelp.js [hunk 4]
```diff
@@ -111,647 +112,778 @@ function IndicesHelp() {
...
+        <InlineMath math={"\\texttt{idxshape:}"} /> Shape parameter of fitted
+        GEV distribution.
+      </p>{" "}
+      <p className="contents">
+        <InlineMath math={"\\texttt{idxscale:}"} /> Scale parameter of fitted
+        GEV distribution.
+      </p>{" "}
+    </Row>
+      {/* impact relevant */}
           <Row className="indices">
```
Comment (raquelalegre) [inline src/components/ExtraPages/Help/IndicesHelp.js]:
```suggestion
    <Row className="indices">
```

**Review by raquelalegre — APPROVED** [review]
Just minor commit suggestions on indentation and commented out line.

**Pull request merged**